CMakeLists.txt

```diff
@@ -93,10 +97,17 @@ endif (${CMAKE_SYSTEM_PROCESSOR} MATCHES "^arm.*$")
 #
 if (MACOSX)
   set (CMAKE_INSTALL_LIBDIR "lib")
+  set (CMAKE_INSTALL_BINDIR "/sbin")
 else (MACOSX) # --> Linux
-  if (DEBIAN)
+  if (DEBIAN OR ARCHLINUX)
+    if (ARCHLINUX)
+      set (CMAKE_INSTALL_BINDIR "/usr/bin")
+    else (ARCHLINUX)
+      set (CMAKE_INSTALL_BINDIR "/sbin")
+    endif (ARCHLINUX)
     set (CMAKE_INSTALL_LIBDIR "lib")
   else (DEBIAN) # --> RedHat, Fedora, CentOS, SuSE
```

Comment:
For some obscure reason CMake wants the same condition to be repeated in `else` and `endif`, please add your `OR ARCHLINUX` here.